commit d77c0899db92dfc2bed535efb91cf9918478fd86
Author: Richard Henderson <rth@twiddle.net>
Date:   Wed Apr 16 21:32:27 2014 -0700

    alpha: Enable unwind tables for backtrace.c

 2014-04-16  Richard Henderson  <rth@redhat.com>
 
+	* sysdeps/alpha/Makefile [debug] (CFLAGS-backtrace.c): Enable
+	unwind tables.
+
 
diff --git a/sysdeps/alpha/Makefile b/sysdeps/alpha/Makefile
index 1cf77fb..bf133b9 100644
--- a/ports/sysdeps/alpha/Makefile
+++ b/ports/sysdeps/alpha/Makefile
@@ -20,6 +20,11 @@ ifeq ($(subdir),db2)
 CPPFLAGS += -DHAVE_SPINLOCKS=1 -DHAVE_ASSEM_ALPHA=1
 endif
 
+ifeq ($(subdir),debug)
+# Consider making this GCC's default...
+CFLAGS-backtrace.c = -fasynchronous-unwind-tables
+endif
+
 ifeq ($(subdir),gmon)
 sysdep_routines += _mcount
 endif
